dashboards-observability/common/constants/shared.ts

```diff
@@ -27,7 +27,7 @@ export const PPL_ENDPOINT = '/_plugins/_ppl';
 export const SQL_ENDPOINT = '/_plugins/_sql';
 export const DSL_ENDPOINT = '/_plugins/_dsl';
 
-export const observabilityID = 'observability';
+export const observabilityID = 'observability-dashboards';
```

Comment:
Why is this one observability-dashboards instead of dashboard-observability??

Reply:
just to be consistent with other plugins. I think part of the reason is that it shows up in URL, which means it has to be a dashboards frontend plugin, but I don't think we have a standard on this